galsim/wfirst/wfirst_detectors.py

```diff
@@ -96,7 +96,11 @@ def applyPersistence(img, prev_exposures):
     @param prev_exposures    List of up to 8 Image instances in the order of exposures, with the
                              recent exposure being the first element.
     """
-    img.applyPersistence(img_list,wfirst.persistence_coefficients)
+    if hasattr(prev_exposures,'__iter__'):
+      n_exp = min(len(prev_exposures),len(galsim.wfirst.persistence_coefficients))
+      img.applyPersistence(prev_exposures[:n_exp],galsim.wfirst.persistence_coefficients[:n_exp])
```

Comment:
I think it makes sense for ```applyPersistence``` to raise an exception if the lengths of those lists do not match.  There is no way for that routine to come up with a reasonable way of guessing whether the user made some kind of mistake or whether there is some reasonable way it should be using that information (or if so, how).  So I would advocate for not changing that.